relay-dynamic-config/src/defaults.rs

```diff
@@ -106,6 +106,17 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
             ),
             tags: Default::default(),
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/http.decoded_response_body_length@byte".into(),
```

Comment:
@DominikB2014 do you still want to use the old MRI here, or rather make a clean cut and update the MRI as well? If I understand correctly, we did not collect any data for the old MRI yet.

Reply:
We shouldn't have collected data unless someone manually instrumented spans using the old span conventions

Reply:
I'd prefer updating the MRI in that case